content/usgs-packages/dataRetrieval_readNWIS.Rmd

```diff
@@ -284,17 +300,16 @@ makeArgDefTable("readNWISpeak", table.num=7)
 
 **Peak flow values for a site near Cassia, Florida.**
 
-The default settings will return data where the date of the peak flow is known. To include peak flows with unknown dates, change `asDateTime` to `FALSE`. This will keep all rows of the data.
+The default settings will return data where the date of the peak flow is known. To see peak flows with incomplete dates, change `convertType` to `FALSE`. 
```

Comment:
yes. `convertType=FALSE` makes it so only characters are returned (including things that might normally come back as numeric).